task5/MyProject/src/Name.java

```diff
@@ -0,0 +1,15 @@
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/get")
+public class Name extends HttpServlet {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String name = request.getParameter("name");
+        if (name.length() <= 100){
```

Comment:
(code style) Not an error, but usually another approach is used, named "early fail/ early return"
It means that instead of 
```
if (something) {
    doSomething();
    doSomething();
    doDomething();
}
```
you write:
```
if (!something) {
   return; 
}
doSomething() ....
```
Early return means less code in brackets. Less code in brackets is easier to read.